ui/src/data/source.tsx

```diff
@@ -2237,7 +2237,7 @@ function fromAPIDisplayHint(hint?: tanagra.DisplayHint): HintData {
         const value = dataValueFromLiteral(enumHintValue.enumVal.value);
         return {
           value,
-          name: enumHintValue.enumVal.display ?? String(value),
+          name: enumHintValue.enumVal.display ?? String(value ?? "n/a"),
```

Comment:
I think it would be better to index the `n/a` string into the hints. That way it could be configurable per entity eventually and would get sorted properly in the list. It feels better to me to make the hints fully consistent but the UI could potentially do it here too.

Reply:
done

Reply:
update: i reverted this and added the value in backend before returning response to UI.